framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1028,10 +1029,15 @@ public void postProcessClassTree(ClassTree tree) {
         TypesIntoElements.store(processingEnv, this, tree);
         DeclarationsIntoElements.store(processingEnv, this, tree);
         if (checker.hasOption("infer") && wholeProgramInference != null) {
-            // Write scenes into .jaif files. In order to perform the write
-            // operation only once for each .jaif file, the best location to
+            // Write out the results of whole-program inference.
+            // In order to perform the write operation only once for each class, the best location
+            // to
             // do so is here.
-            wholeProgramInference.saveResults();
+            WholeProgramInference.OutputKind outputKind =
+                    checker.hasOption("outputStubs")
```

Comment:
I would change `-AoutputStubs` to `-Ainfer=stubs`. Then this would be `checker.getOption("infer").equals("stubs")`.   That makes documenting this option easier.